libs/util/git/src/git-client.service.ts

```diff
@@ -282,29 +281,41 @@ export class GitClientService {
       baseBranch,
     });
 
-    const { diff } = await this.preCommitProcess({
+    // calculate diff on everything *already existing* on the amplication branch since last amplication build
+    const preCommitDiff = await this.preCommitProcess({
```

Comment:
Since the precommit method is now on charge of 2 operations, would it make more sense to rename it in something more specific?